source/community/events/archives/workshop/workshop-board-meetings.html.md

```diff
@@ -13,10 +13,8 @@ The purpose of this page is to collaboratively plan oVirt Board Meetings at [OVi
 
 ## Upcoming Workshops Where a Board Meeting May Be Held
 
-*   August 28, 2012: LinuxCon North America
-*   November 7-9, 2012: LinuxCon Europe (workshop will be co-located with KVM Forum)
-*   January 22-24, 2013: NetApp Headquarters in Sunnyvale, California, US
-*   March 20-21, 2013: Intel Campus in Shanghai, China
+*   Los Angeles, September 11-13, 2017: [LinuxCon North America](http://events.linuxfoundation.org/events/linuxcon-north-america)
```

Comment:
@mykaul Suggest "Los Angeles, September 11-13, 2017: Open Source Summit North America (The new name for LinuxCon)."